custom_components/grocy/__init__.py

```diff
@@ -31,9 +35,15 @@
     REQUIRED_FILES,
     STARTUP,
     VERSION,
+    STOCK_NAME,
+    CHORES_NAME,
+    SHOPPING_LIST_NAME,
+    EXPIRING_PRODUCTS_NAME,
+    EXPIRED_PRODUCTS_NAME,
+    MISSING_PRODUCTS_NAME,
 )
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=300)
```

Comment:
Is this related to adding config flow?

Reply:
Yes and no. Since add-on is not up to data, we can not check if grocy db was updated.
With 30 seconds and all sensors, grocy add-on cpu usage is high asf...